Open Recent is broken on Windows (wrong path separator?)

File menu -> Open Recent -> entries are listed as eg `/D:/Meshroom_Dev/test-project/mostree.mg`

Clicking on an entry results in:

```
[2020-05-23 16:12:48,660][ERROR] Traceback (most recent call last):
  File "D:\Meshroom_Src\meshroom\meshroom\ui\reconstruction.py", line 432, in load
    super(Reconstruction, self).load(filepath, setupProjectFile)
  File "D:\Meshroom_Src\meshroom\meshroom\ui\graph.py", line 314, in load
    g.load(filepath, setupProjectFile)
  File "D:\Meshroom_Src\meshroom\meshroom\core\graph.py", line 247, in load
    with open(filepath) as jsonFile:
OSError: [Errno 22] Invalid argument: '/D:/Meshroom_Dev/test-project/mostree.mg'
```

Path should be `D:\Meshroom_Dev\test-project\mostree.mg`.

Exception dialog can be closed, and the program is still usable; can manually open the project using file->open.

--- a/meshroom/ui/app.py
+++ b/meshroom/ui/app.py
@@ -184,7 +184,7 @@ class MeshroomApp(QApplication):
 
     @Slot(str)
     def addRecentProjectFile(self, projectFile):
-        projectFile = QUrl(projectFile).path()
+        projectFile = QUrl(projectFile).toLocalFile()
         projects = self._recentProjectFiles()
 
         # remove duplicates while preserving order

--- a/meshroom/ui/qml/Viewer/FeaturesInfoOverlay.qml
+++ b/meshroom/ui/qml/Viewer/FeaturesInfoOverlay.qml
@@ -83,7 +83,6 @@ FloatingPane {
                     checked: true
                     text: MaterialIcons.center_focus_strong
                     onClicked: {
-                        console.warn("featuresVisibilityButton.checked: " + featuresVisibilityButton.checked)
                         featureType.viewer.displayfeatures = featuresVisibilityButton.checked;
                     }
                     font.pointSize: 10
@@ -97,7 +96,6 @@ FloatingPane {
                     checked: true
                     text: MaterialIcons.timeline
                     onClicked: {
-                        console.warn("tracksVisibilityButton.checked: " + tracksVisibilityButton.checked)
                         featureType.viewer.displayTracks = tracksVisibilityButton.checked;
                     }
                     font.pointSize: 10
@@ -110,7 +108,6 @@ FloatingPane {
                     checked: true
                     text: MaterialIcons.fiber_manual_record
                     onClicked: {
-                        console.warn("landmarksVisibilityButton.checked: " + landmarksVisibilityButton.checked)
                         featureType.viewer.displayLandmarks = landmarksVisibilityButton.checked;
                     }
                     font.pointSize: 10

--- a/meshroom/ui/qml/Viewer/Viewer2D.qml
+++ b/meshroom/ui/qml/Viewer/Viewer2D.qml
@@ -392,7 +392,6 @@ FocusScope {
                                 active = true;
                         }
                         onActiveNodeChanged: {
-                            console.warn("mtracksLoader.onActiveNodeChanged, activeNode: " + activeNode)
                             if(!isUsed)
                                 active = false;
                             else if(!isComputed)

